Modules/selectmodule.c

```diff
@@ -2148,37 +2148,36 @@ kqueue_queue_control(kqueue_queue_Object *self, PyObject *args)
     }
 
     if (ch != NULL && ch != Py_None) {
-        it = PyObject_GetIter(ch);
-        if (it == NULL) {
-            PyErr_SetString(PyExc_TypeError,
-                            "changelist is not iterable");
+        seq = PySequence_Fast(ch, "changelist is not iterable");
+        if (seq == NULL) {
             return NULL;
         }
-        nchanges = PyObject_Size(ch);
-        if (nchanges < 0) {
+        if (PySequence_Fast_GET_SIZE(arg) > INT_MAX / sizeof(struct kevent)) {
```

Comment:
`arg` should be `seq`, right?

I don't think we need to `/ sizeof(struct kevent)`. `> INT_MAX` is enough.